src/main/java/frc/swerve/Wheel.java

```diff
@@ -62,15 +63,21 @@ public void disable() {
 
 	public void set(double radians, double speed) {
 		if (speed != 0.0) {
+
 			double targetPosition = radians / TWO_PI;
+			// make target position between 0 and 1.0
 			targetPosition = GRTUtil.positiveMod(targetPosition, 1.0);
 
+			// get the current position
 			int encoderPosition = rotateMotor.getSelectedSensorPosition(0) - OFFSET;
 			double currentPosition = encoderPosition / TICKS_PER_ROTATION;
 			double rotations = Math.floor(currentPosition);
```

Comment:
Why are using `floor()` to create an integer and then storing it in a `double` value?